.eslintrc

```diff
@@ -1,13 +1,14 @@
 {
   "parserOptions": {
-    "ecmaVersion": 6,
-    "sourceType": "module"
+    "ecmaVersion": 2018, // suppport syntax features (like Object spread operator) up to es2018
```

Comment:
Maybe use the latest version 2021?

Reply:
Unfortunately, we can’t use es2021 yet because we have to support node 10 for Stapler (it uses stencil-styles) and node 10 supports ES features up to 2018 only